packages/entity-browser/src/modules/listView/sagas.js

```diff
@@ -55,7 +57,10 @@ export function* fetchEntitiesAndAddToStore(page) {
   if (!entityStore[page]) {
     const searchInputs = yield call(getSearchInputs)
 
-    const entities = yield call(api.fetchEntities, entityName, page, orderBy, limit, columnDefinition, searchInputs)
+    const fields = _union(...columnDefinition.map(field => (field.value)))
```

Comment:
const fields = columnDefinition.map(field => (field.value))
would get you an array of arrays with the fields. but i have rewritten it a bit and removed the duplicate check since REST API should handle that well.